### ISSUE TITLE
Tutorial update, fix for Mantis 1.2.xx, bug on mantis 1.3.3

### ISSUE THREAD

**Issue by tanidar** [nikeyes/Mantis2Trello#2]
Hello,

It would be nice if you add the mention into a tutorial that Mantis installation must be in /mantis/ or /bugs/   subfolder to get the plugin work.

Also there are the problems with Mantis 1.2.xx installations, plugin returns error into console and no card is created.

```
Error in response to storage.get: TypeError: Cannot read property 'innerText' of null
    at createCard (chrome-extension://ddiiaojmncfmdmghfinjfedilkegilom/content.js:39:108)
    at Object.authenticationSuccess [as success] (chrome-extension://ddiiaojmncfmdmghfinjfedilkegilom/content.js:12:5)
    at Object.authorize (chrome-extension://ddiiaojmncfmdmghfinjfedilkegilom/client.js:6:158)
    at Object.authorizationTrello [as callback] (chrome-extension://ddiiaojmncfmdmghfinjfedilkegilom/content.js:106:12)
    at onMessageCallback (chrome-extension://ddiiaojmncfmdmghfinjfedilkegilom/content.js:120:23)
```

**Comment by tanidar** [comment]
There is another issue: on Mantis 1.3.3 there is an exception in console when plugin is trying to add the Trello URL link into bug notes

```
content.js:26 Uncaught TypeError: Cannot read property 'submit' of undefined
    at Object.creationSuccess [as success] (chrome-extension://ddiiaojmncfmdmghfinjfedilkegilom/content.js:26:48)
    at n (chrome-extension://ddiiaojmncfmdmghfinjfedilkegilom/jquery.1.7.1.min.js:2:14784)
    at Object.fireWith [as resolveWith] (chrome-extension://ddiiaojmncfmdmghfinjfedilkegilom/jquery.1.7.1.min.js:2:15553)
    at w (chrome-extension://ddiiaojmncfmdmghfinjfedilkegilom/jquery.1.7.1.min.js:4:11395)
    at XMLHttpRequest.d (chrome-extension://ddiiaojmncfmdmghfinjfedilkegilom/jquery.1.7.1.min.js:4:17218)`
```

**Comment by nikeyes** [comment]
Hi tanidar,

In your Mantis, have you permissions to modify  the ticket?

The extensión only work if you have edit permissions over the ticket.

Sorry for the inconvenient.

**Comment by tanidar** [comment]
Hello nikeyes,

Yes I have permissions, actually I'm the root administrator of this mantis instance. The comment is successfully written into text input field but plugin is failed to submit bug note form

**Comment by nikeyes** [comment]
Solved in the 0.2 version.
